modules/mysql/main.tf

```
@@ -27,8 +27,9 @@ locals {
   users     = { for u in var.additional_users : u.name => u }
   iam_users = {
     for user in var.iam_users : user.id => {
-      email         = user.email,
+      email         = user.email
       is_account_sa = trimsuffix(user.email, "gserviceaccount.com") == user.email ? false : true
+      type          = user.type != null ? user.type : null
```

nit: Should we remove `is_account_sa` altogether since it is utlimately used for finding out type only!
```suggestion
      type          =  trimsuffix(user.email, "gserviceaccount.com") == user.email ? (user.type != null ? user.type : "CLOUD_IAM_USER") : "CLOUD_IAM_SERVICE_ACCOUNT"
```